### PR TITLE
Fix mako no detecting add|remove of root folders. Populate with JS

### PR DESCRIPTION
**Pull request by fernandog**
@OmgImAlexis

Mako loads once the code in the import, so it won't see that app.ROOT_DIRS changed

### CODE REVIEW

File: static/js/home/index.js [hunk 1]
```diff
@@ -349,4 +349,27 @@ MEDUSA.home.index = function() {
...
+    if (backend_dirs.length > 1) {
+        $('#showRoot').show()
+        var item = ['All Folders'];
+        rootDirOptions = item.concat(backend_dirs);
+        $.each(rootDirOptions, function (i, item) {
+            $('#showRootDir').append($('<option>', {
+                value: i - 1,
+                text : item,
+            }));
+        $('select#showRootDir').prop('selectedIndex', rootDirIndex + 1);
```
Comment (OmgImAlexis) [inline static/js/home/index.js]: Please indent this.

File: static/js/home/index.js [hunk 2]
```diff
@@ -349,4 +349,27 @@ MEDUSA.home.index = function() {
             log.info(err);
         });
     });
+
+    var rootDir = MEDUSA.config.rootDirs
+    var rootDirIndex = MEDUSA.config.showsRoot.id
+    if (rootDir) {
+        backend_pieces = rootDir.split('|')
+        backend_default = 'rd-' + backend_pieces[0]
+        backend_dirs = backend_pieces.slice(1)
```
Comment (OmgImAlexis) [inline static/js/home/index.js]: Vars need to be camelCase, please stop using snake_case.

File: medusa/server/api/v2/config.py [hunk 3]
```diff
@@ -129,6 +129,9 @@ def get(self, query=''):
                     'allSeasons': bool(app.DISPLAY_ALL_SEASONS),
                     'specials': bool(app.DISPLAY_SHOW_SPECIALS)
                 }
+            },
+            'showsRoot': {
+                'id': int(app.SHOWS_ROOT) if app.SHOWS_ROOT else None
```
Comment (OmgImAlexis) [inline medusa/server/api/v2/config.py]: Is there a reason not to just return all the show roots dirs with their id?

Comment (fernandog) [inline medusa/server/api/v2/config.py]: I only need id

Comment (OmgImAlexis) [inline medusa/server/api/v2/config.py]: I know YOU only need the id but is there a reason not to just return the whole show root object/string? It'd make more sense for the API to expose it all. Think about API(third party) first before how Medusa uses it.

File: static/js/home/index.js (hunk 1 above)
Comment (OmgImAlexis) [inline static/js/home/index.js]: @fernandog this should be inline with the line above..

File: static/js/home/index.js [hunk 4]
```diff
@@ -348,17 +346,16 @@ MEDUSA.home.index = function() {
...
     if (rootDir) {
-        backend_pieces = rootDir.split('|')
-        backend_default = 'rd-' + backend_pieces[0]
-        backend_dirs = backend_pieces.slice(1)
+        backendPieces = rootDir.split('|');;
+        backendDirs = backendPieces.slice(1)
     }
-    if (backend_dirs.length > 1) {
-        $('#showRoot').show()
+    if (backendDirs.length > 1) {
```
Comment (OmgImAlexis) [inline static/js/home/index.js]: This should be `>= 1` right?

Comment (fernandog) [inline static/js/home/index.js]: no. we only show if users has TWO root dirs

**Pull request merged**